src/background/tab-manager.ts

```diff
@@ -79,6 +79,23 @@ export default class TabManager {
                 a.download = name;
                 a.click();
             }
+            if (type === 'export-css-proxy') {
+                const tabs = await queryTabs({});
+                const activeURL = await this.getActiveTabURL();
+                tabs.filter((tab) => this.ports.has(tab.id))
+                    .filter((tab) => tab.url === activeURL)
+                    .forEach((tab) => {
+                        const framesPorts = this.ports.get(tab.id);
+                        framesPorts.forEach(({port}, frameId) => {
+                            const message = {type: 'export-css'};
+                            if (tab.active && frameId === 0) {
+                                port.postMessage(message);
+                            } else {
+                                setTimeout(() => port.postMessage(message));
```

Comment:
It should not do anything for iframes. Users are supposed to use it for their pages, but will download CSS for iframes like YouTube etc.